lua/telescope/entry_manager.lua

```diff
@@ -65,18 +78,28 @@ function EntryManager:get_container(index)
   return {}
 end
 
+--- Get the entry in the EntryManager corresponding to `index`
+---@param index number
+---@return table: table with information about the given entry
 function EntryManager:get_entry(index)
   return self:get_container(index)[1]
 end
 
+--- Get the score in the EntryManager corresponding to `index`
+---@param index number
+---@return number
 function EntryManager:get_score(index)
   return self:get_container(index)[2]
 end
 
+--TODO: documentation
 function EntryManager:get_ordinal(index)
   return self:get_entry(index).ordinal
 end
 
+--- Get the index of the given entry in the EntryManager
+---@param entry table: table with information about the given entry
+---@return integer/nil: the index of the entry if is present, nil otherwise
```

Comment:
About emmy documentation. You can have multiple types for `@param` but not for `@return` seems like an oversight. Will open an issue, maybe i implement it but if i do we will end up with `---@return number|nil` (to keep consistency with `@param`) (there is no integer type in emmy doc its always number)